### PR TITLE
3h2a exception clarification

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
See Issue #826

### CODE REVIEW

**Review by lgarron — APPROVED** [review]

**Comment by hanwu85** [comment]
Why don't we use "a logo sticker"?   Simply "a logo" could be confusing because from the wording "a logo" is not a transparent part so why does it need to be listed in the exception?

**Comment by Goosly** [comment]
So this means stickerless, non-transparent cubes with transparent stickers on all sides are forbidden? Or is a sticker not a part?

**Comment by Ivan-Brigidano** [comment]
> Why don't we use "a logo sticker"?

@hanwu85 I believe it's better to use "logo" to make it uniform with the rest of regulations.

> Simply "a logo" could be confusing because from the wording "a logo" is not a transparent part so why does it need to be listed in the exception?

@hanwu85 I don't agree with you. Usually logos are transparent but some letters or pictures. So most logos are transparent.

> So this means stickerless, non-transparent cubes with transparent stickers on all sides are forbidden? Or is a sticker not a part?

@Goosly this exception is right now only forbidden logos and this change is just a clarification.

**Comment by hanwu85** [comment]
> > Why don't we use "a logo sticker"?
> 
> I believe it's better to use "logo" to make it uniform with the rest of regulations.

I think it's a "exception", it means a sort of example, so it's not necessary to be a uniformed terminology.  You need to let people understand, that's why you list it under  "exception".

> > Simply "a logo" could be confusing because from the wording "a logo" is not a transparent part so why does it need to be listed in the exception?
> 
>  I don't agree with you. Usually logos are transparent but some letters or pictures. So most logos are transparent.

This time, I don't agree with you.  Some logos are printed on the plastic, some logos are on a white colored sticker or other colored stickers, and some comes with logo on an transparent sticker.  So I think simply "a logo" could be confusion,  I don't mean it's wrong, I mean it could cause confusion from interpretation of the Regulations.

**Comment by Ivan-Brigidano** [comment]
@hanwu85 logos are covered by [3l](https://www.worldcubeassociation.org/regulations/#3l) so people shouldn't be confused about it. Let's wait to see other WRC members opinion about it!

**Comment by hanwu85** [comment]
For the record,  I think "a logo" is okay but just not clear enough to eliminate confusion when reading this particular one.

**Comment by Laura-O** [comment]
I agree with @hanwu85.
Taking the regulation change literally, it does forbid transparent overlay sticker with a printed black logo (like the ones used by Moyu).

**Comment by Ivan-Brigidano** [comment]
For me it's pretty clear. But a WRC member and a Senior Delegate don't think so. Maybe something like `An overlay logo sticker` could work

**Comment by xsrvmy** [comment]
We still need to address the GAN X problem, and also SCS coating

**Comment by hanwu85** [comment]
I am not familiar with the GAN X problem, what is it?

**Comment by Ivan-Brigidano** [comment]
@xsrvmy this Pull Request is for clarify the exception. The discussion for the clarification of transparent cubes please write it here #736

**Review by hanwu85 — APPROVED** [review]

File: wca-regulations.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 3h) Modifications that enhance the basic concept of a puzzle are not permitted. Modified versions of puzzles are permitted only if the modification does not make any additional information available to the competitor (e.g. orientation or identity of pieces), compared to an unmodified version of the same puzzle.
     - 3h1) "Pillowed" puzzles are permitted.
     - 3h2) Puzzles whose colored plastic is visible inside the puzzle (e.g. "stickerless" puzzles) are permitted. This does not include the following puzzles:
-        - 3h2a) Puzzles with transparent parts. Exception: An overlay sticker (see [Regulation 3l](regulations:regulation:3l)).
+        - 3h2a) Puzzles with transparent parts. Exception: An overlay sticker or a logo (see [Regulation 3l](regulations:regulation:3l)).
```
Comment (lgarron) [inline wca-regulations.md]:
```suggestion
        - 3h2a) Puzzles with transparent parts that can reveal more information about the state of the puzzle. Exception: An overlay sticker or a logo (see [Regulation 3l](regulations:regulation:3l)).
```

Comment (lgarron) [reply wca-regulations.md]: To help clarify that overlay stickers like https://www.worldcubeassociation.org/incidents/42 are allowed.

**Review by lgarron — APPROVED** [review]

**Pull request merged**